### PR TITLE
Add jobs to test and push relase-blocker image

### PR DESCRIPTION
**Pull request by fgimenez**
/cc @rmohr @dhiller 

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

**Review by rmohr — APPROVED** [review]
/lgtm
/approve

File: github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml [hunk 1]
```diff
@@ -27,6 +26,32 @@ postsubmits:
...
+        testgrid-create-test-group: "false"
+      decorate: true
+      cluster: ibm-prow-jobs
+      max_concurrency: 1
+      labels:
+        preset-bazel-cache: "true"
+        preset-kubevirtci-quay-credential: "true"
+      spec:
+        containers:
+          - image: quay.io/kubevirtci/bootstrap:v20210126-a12b6c0
```
[the remark this answers is not in the document]
Comment (fgimenez) [reply github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml:42]: Nope, copied from the rehearse job. I think it is good though for the purposes of this job, just bazelisk in place is enough.

**Comment by fgimenez** [comment]
/hold

waiting for a full successful build on CI

**Comment by fgimenez** [comment]
/unhold

full successful execution here https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-release-blocker-image/1420415782294130688

File: github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fgimenez) [reply github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml:42]: I tried to bump bootstrap image and it failed https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-release-blocker-image/1420410039285321728 I think we already saw similar errors previously when trying to use a fedora-based bootstrap image on jobs that use docker rules, will work on it in a follow-up.

**Pull request merged**